tracee-ebpf/main.go

```diff
@@ -26,6 +26,7 @@ var buildPolicy string
 
 // These vars are supposed to be injected at build time
 //go:embed "dist/tracee.bpf/*"
+//go:embed "dist/tracee.bpf.core.o"
```

Comment:
the makefile target is customizable (user can choose the name of the artifact) but the go code hardcodes the default name. It's really not important for anyone so I don't think we should do anything different now (I have some ideas for the future) just making the note here in context for future reference.